hack/make/dependencies.make

```diff
@@ -34,39 +35,42 @@ CLUSTERADM_VERSION ?= 0.6.0
 SUBCTL_VERSION ?= release-0.15
 GINKGO_VERSION ?= v2.6.1
 
+.PHONY: dependencies
+dependencies: kustomize operator-sdk controller-gen envtest kind helm yq istioctl clusteradm subctl ginkgo
+	@echo "dependencies installed successfully"
```

Comment:
Would it make sense to include the update to PATH here?

Reply:
e.g. `export PATH=$PATH:$(pwd)/bin`

Reply:
or even an echo to recommend it if it's not already done.

Reply:
I don't think we want to do that. Do we tell people somewhere to do that?

Reply:
I suppose some extra complexity if we export ourselves (other than potentially clobbering existing paths if they've got some of these installed already) is that we'd need to cater for a bunch of shells (`zsh`, `bash`).

We could output a path to include in their shell of choice at the end of local-setup?

Reply:
` Do we tell people somewhere to do that?` we don't but some of the individual commands did have an output to say consider doing it. I guess that's standard to make sure the binary is available.